.mergify.yml

```diff
@@ -227,16 +227,16 @@ pull_request_rules:
       - "check-success=encryption-pvc-kms-vault-k8s-auth"
       - "check-success=lvm-pvc"
       - "check-success=rgw-multisite-testing"
-      - "check-success=TestCephSmokeSuite (v1.22.17)"
-      - "check-success=TestCephSmokeSuite (v1.28.0)"
-      - "check-success=TestCephHelmSuite (v1.22.17)"
-      - "check-success=TestCephHelmSuite (v1.28.0)"
-      - "check-success=TestCephMultiClusterDeploySuite (v1.28.0)"
-      - "check-success=TestCephObjectSuite (v1.27.2)"
-      - "check-success=TestCephUpgradeSuite (v1.22.17)"
-      - "check-success=TestCephUpgradeSuite (v1.28.0)"
-      - "check-success=TestHelmUpgradeSuite (v1.22.17)"
-      - "check-success=TestHelmUpgradeSuite (v1.28.0)"
+      - "check-success=TestCephSmokeSuite (v1.23.17)"
```

Comment:
Revert this file, these versions won't apply to the 1.12 branch since we're not backporting this.

Reply:
reverted